letsencrypt/client/standalone_authenticator.py

```diff
@@ -0,0 +1,519 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""An authenticator that doesn't rely on any existing server program,
```

Comment:
Docstrings should be maintained in the following convention:
```
"""Short one line description.

More desciscrpiton,
possibly multiline.

"""
```
The autogenerrated docs is much more readable this way.